projects/glib/build.sh

```diff
@@ -21,49 +21,18 @@ rm -rf $BUILD
 mkdir $BUILD
 
 meson $BUILD \
+  -Db_lundef=false \
   -Ddefault_library=static \
   -Dlibmount=false \
   -Dselinux=false
 
 ninja -C $BUILD
 
-$CC $CFLAGS -I. -Iglib -I$BUILD/glib -c $SRC/fuzz_bookmark.c
-$CXX $CXXFLAGS -lFuzzingEngine \
-  fuzz_bookmark.o -o $OUT/fuzz_bookmark \
-  $BUILD/glib/libglib-2.0.a $BUILD/glib/libcharset/libcharset.a
-cp $SRC/fuzz.options $OUT/fuzz_bookmark.options
-find glib/tests -type f -size -32k -name "*.xbel" \
-  -exec zip -qju $OUT/fuzz_bookmark_seed_corpus.zip "{}" \;
+find $BUILD/fuzzing -maxdepth 1 -executable -type f -exec cp "{}" $OUT \;
 
-$CC $CFLAGS -I. -Iglib -I$BUILD/glib -c $SRC/fuzz_key.c
-$CXX $CXXFLAGS -lFuzzingEngine \
-  fuzz_key.o -o $OUT/fuzz_key \
-  $BUILD/glib/libglib-2.0.a $BUILD/glib/libcharset/libcharset.a
-cp $SRC/fuzz.options $OUT/fuzz_key.options
-find gio/tests -type f -size -32k -name "*.desktop" \
-  -exec zip -qju $OUT/fuzz_key_seed_corpus.zip "{}" \;
+find fuzzing -type f -name "*.dict" -exec cp "{}" $OUT \;
```

Comment:
Should it be `$BUILD/fuzzing` instead of just `fuzzing` like on the line 31?

Reply:
Those files aren't put in `$BUILD`.